.github/workflows/ci.yml

```diff
@@ -19,6 +19,23 @@ env:
   REGISTRY: ghcr.io
 
 jobs:
+  rustfmt:
+    concurrency:
```

Comment:
Do we need job level concurrency settings here? Seems like this is superseded by the workflow-level concurrency setting on line 12.